runtime/browser/android/cookie_manager.cc

```diff
@@ -7,28 +7,43 @@
 #include <string>
 
 #include "android_webview/browser/scoped_allow_wait_for_legacy_web_view_api.h"
```

Comment:
We shouldn't include the file from `android_webview`. Because this component is not built into crosswalk